### DISCUSSION TITLE
Does Ozone s3 support Trash?

### DISCUSSION THREAD

**Discussion by tolstykh-da** [apache/ozone#4860] [General]
trino runs through org.apache.hadoop.fs.s3a.S3AFileSystem
it successfully creates tables, but when deleted, the table does not go to the trash. 
and we don’t see the basket folder itself, for example, through a duck

**Comment by sadanand48** [comment]
Hi @tolstykh-da , Ozone trash is only supported for filesystem clients that operate on Filesystem/Legacy buckets . There was a recent [change ](https://issues.apache.org/jira/browse/HDDS-8547)done to support deletions through` ozone sh` to use the trash but that too is specific to FSO buckets. It is more so a filesystem feature.

**Reply by tolstykh-da** [reply]
@sadanand48 Hi, can I ask the community to start a request that the trash would work through org.apache.hadoop.fs.s3a.S3AFileSystem for s3g?
How to live without trash is not clear, especially with dwh
It makes us very sad
